### ISSUE TITLE
JPype is incompatible with Python 3.7: Fatal Python error: _Py_InitializeCore: main interpreter already initialized

### ISSUE THREAD

**Issue by vstinner** [originell/jpype#331]
Since Python 3.7, calling Py_Initialize() twice now fails with a fatal error:

```
Fatal Python error: _Py_InitializeCore: main interpreter already initialized
```

But PyInit__jpype() calls Py_Initialize() at native/python/jpype_python.cpp:161:

https://github.com/originell/jpype/blob/490eb6511a0a0b57be2191cbbf3a90bac45e5879/native/python/jpype_python.cpp#L150-L157

It doesn't make sense to call Py_Initialize() in the initialization function of a module, since Python must be initialized to able to... import jpype :-)

The fix is just to remove the call.

**Comment by vstinner** [comment]
Note: I found the issue when running the test suite of https://github.com/konlpy/konlpy with Python 3.7.

**Comment by marscher** [comment]
Thanks! good catch. Are you willing to remove this line with Githubs
online editor and send us a PR? :)

**Comment by vstinner** [comment]
> Thanks! good catch. Are you willing to remove this line with Githubs online editor and send us a PR? :)

I'm actually trying to do that.

**Comment by marscher** [comment]
there is the edit button hidden somewhere in the right upper corner :)

**Comment by vstinner** [comment]
cc @hroncok

**Comment by rapgro** [comment]
FTC, [PEP 432](https://www.python.org/dev/peps/pep-0432/)

**Comment by Thrameos** [comment]
Yes.  That is one of the lines that caused PyPy to fail.  Unfortunately, there were additional issues so I never go around to patching it and didn't look at the PEP to see if it belonged.  Thanks for catching it.

**Comment by mludvig** [comment]
Hi are you going to release the fixes package anytime soon? I ran into this problem today and would very much prefer to use an official release from PyPy rather than a patched one. Thanks! :)

**Comment by vstinner** [comment]
My PR #332 has been merged, and in the meanwhile, Python 3.7.1 has been released. Since Python 3.7.1, calling Py_Initialize() twice is accepted again ;-)
